src/util/records.py

```diff
@@ -546,6 +556,27 @@ def remove_group(cr, xml_id=None, group_id=None):
 
 
 def rename_xmlid(cr, old, new, noupdate=None, on_collision="fail"):
+    """
+    Rename a :term:`external Identifier` of a record.
```

Comment:
Are terms case sensitive?
```suggestion
    Rename a :term:`external identifier` of a record.
```